### PR TITLE
edit

### PR DESCRIPTION
**Pull request by bossidy**

### CODE REVIEW

**Comment by bossidy** [comment]
please review

**Comment by bossidy** [comment]
"The discovery workflow can be updated to do additional work or steps for the installation of RackHD, to run other workflows based on the SKU analysis, or perform other actions based on the logic embedded into the workflow itself."

Can "additional steps FOR THE INSTALLATION OF RACKHD" be correct?  RackHD is already up and running.

**Comment by heckj** [comment]
My bad - it's not "for the installation of RackHD", but more generally "to do additional work or steps for the installation of an OS, ..."

File: docs/how_it_works.rst [hunk 1]
```diff
@@ -47,18 +47,16 @@ these as *passive* and *active* discovery.
   This is enabled by making a post to the REST interface that RackHD can then add to its data model.
 
 * Active discovery is invoked when a machine attempts to PXE boot on the network that RackHD is
-  monitoring. As a new machine PXE boots, RackHD retrieves the MAC address of the machine.
+  monitoring. As a machine PXE boots, RackHD retrieves the MAC address of the machine.
   If the MAC address has not been recorded, RackHD creates a new record in the data model and
-  then invokes a default workflow. To enable active discovery, you set the default workflow that
-  will be run when a new machine is identified to one of the discovery workflows included
-  within the system. The most common is the SKU Discovery workflow.
+  then invokes a default workflow. Usually, the SKU Discovery workflow is the default.
```
Comment (heckj) [inline docs/how_it_works.rst]: I was trying to aim at the knowledge that "default workflow" was something configurable on the overall system, as opposed to implying that the default code functionality would do this.

Comment (bossidy) [inline docs/how_it_works.rst]: OK, I'll work on it.

**Comment by bossidy** [comment]
Note: This example RackHD instance does not include complete out-of-band networking control. Running workflows requires a “no-op” OBM management setting on discovered nodes and *manual restarts of VMs*.

Question:  Is 'manual restarts of VMs' actually due to not having obm or is it an unrelated limitation?

**Pull request closed without merging**